### ISSUE TITLE
make format reports errors

### ISSUE THREAD

**Issue by lixinswitchpi** [kscalelabs/ksim-gym-zbot#16]
installed ksim from branch wesley/walking, and run python -m examples.walking without a problem, then check out ksim-gym-zbot, once run make format, got following errors ;(, question is which ksim should I to use? I can see only wesley/walking branch has acator class 

train.py: note: In member "forward" of class "Actor":
train.py:166:18: error: Module has no attribute "MixtureOfGaussians"  [attr-defined]
            dist_n = ksim.MixtureOfGaussians(means_nm=mean_nm, stds_nm=std_nm, logits_nm=logits_nm)
                     ^~~~~~~~~~~~~~~~~~~~~~~
train.py: note: In member "get_physics_randomizers" of class "ZbotWalkingTask":
train.py:558:5: error: Return type "list[PhysicsRandomizer]" of "get_physics_randomizers" incompatible with return type "Mapping[str, PhysicsRandomizer]"
in supertype "ksim.task.rl.RLTask"  [override]
        def get_physics_randomizers(self, physics_model: ksim.PhysicsModel) -> list[ksim.PhysicsRandomizer]:
        ^
train.py: note: In member "get_events" of class "ZbotWalkingTask":
train.py:567:5: error: Return type "list[Event]" of "get_events" incompatible with return type "Mapping[str, Event]" in supertype "ksim.task.rl.RLTask" 
[override]
        def get_events(self, physics_model: ksim.PhysicsModel) -> list[ksim.Event]:
        ^
train.py:569:13: error: Module has no attribute "PushEvent"  [attr-defined]
                ksim.PushEvent(
                ^~~~~~~~~~~~~~
train.py: note: In member "get_observations" of class "ZbotWalkingTask":
train.py:590:49: error: Argument "noise" to "JointPositionObservation" has incompatible type "float"; expected "Noise | None"  [arg-type]
                ksim.JointPositionObservation(noise=math.radians(2)),
                                                    ^~~~~~~~~~~~~~~
train.py:591:49: error: Argument "noise" to "JointVelocityObservation" has incompatible type "float"; expected "Noise | None"  [arg-type]
                ksim.JointVelocityObservation(noise=math.radians(10)),
                                                    ^~~~~~~~~~~~~~~~
train.py:601:13: error: Unexpected keyword argument "lag_range" for "create" of "ProjectedGravityObservation"  [call-arg]
                ksim.ProjectedGravityObservation.create(
                ^
/home/xiaowen/anaconda3/envs/ksim/lib/python3.11/site-packages/ksim/observation.py:319:6: note: "create" of "ProjectedGravityObservation" defined here
train.py:588:5: error: Return type "list[Observation]" of "get_observations" incompatible with return type "Mapping[str, Observation]" in supertype
"ksim.task.rl.RLTask"  [override]
        def get_observations(self, physics_model: ksim.PhysicsModel) -> list[ksim.Observation]:
        ^
train.py:605:23: error: Argument "noise" to "create" of "ProjectedGravityObservation" has incompatible type "float"; expected "Noise | None"  [arg-type]
                    noise=math.radians(1),
                          ^~~~~~~~~~~~~~~
train.py:616:23: error: Argument "noise" to "create" of "SensorObservation" has incompatible type "float"; expected "Noise | None"  [arg-type]
                    noise=1.0,
                          ^~~
train.py:621:23: error: Argument "noise" to "create" of "SensorObservation" has incompatible type "float"; expected "Noise | None"  [arg-type]
                    noise=math.radians(10),
                          ^~~~~~~~~~~~~~~~
train.py: note: In member "get_commands" of class "ZbotWalkingTask":
train.py:625:5: error: Return type "list[Command]" of "get_commands" incompatible with return type "Mapping[str, Command]" in supertype
"ksim.task.rl.RLTask"  [override]
        def get_commands(self, physics_model: ksim.PhysicsModel) -> list[ksim.Command]:
        ^
train.py: note: In member "get_rewards" of class "ZbotWalkingTask":
train.py:628:5: error: Return type "list[Reward]" of "get_rewards" incompatible with return type "Mapping[str, Reward]" in supertype "ksim.task.rl.RLTask"
 [override]
        def get_rewards(self, physics_model: ksim.PhysicsModel) -> list[ksim.Reward]:
        ^
train.py:635:13: error: Module has no attribute "AngularVelocityPenalty"; maybe "ActionVelocityPenalty" or "AngularVelocityReward"?  [attr-defined]
                ksim.AngularVelocityPenalty(index=("x", "y", "z"), scale=-0.005),
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~
train.py:636:13: error: Module has no attribute "LinearVelocityPenalty"; maybe "JointVelocityPenalty", "ActionVelocityPenalty", or "LinearVelocityReward"?
 [attr-defined]
                ksim.LinearVelocityPenalty(index=("x", "y", "z"), scale=-0.005),
                ^~~~~~~~~~~~~~~~~~~~~~~~~~
train.py: note: In member "get_terminations" of class "ZbotWalkingTask":
train.py:654:5: error: Return type "list[Termination]" of "get_terminations" incompatible with return type "Mapping[str, Termination]" in supertype
"ksim.task.rl.RLTask"  [override]
        def get_terminations(self, physics_model: ksim.PhysicsModel) -> list[ksim.Termination]:
        ^
train.py:656:13: error: Unexpected keyword argument "unhealthy_z_lower" for "BadZTermination"  [call-arg]
                ksim.BadZTermination(unhealthy_z_lower=0.05, unhealthy_z_upper=0.5),
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
train.py:656:13: error: Unexpected keyword argument "unhealthy_z_upper" for "BadZTermination"  [call-arg]
                ksim.BadZTermination(unhealthy_z_lower=0.05, unhealthy_z_upper=0.5),
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
train.py: note: In member "get_model" of class "ZbotWalkingTask":
train.py:669:25: error: Argument 1 of "get_model" is incompatible with supertype "xax.task.mixins.train.TrainMixin"; supertype defines the argument type
as "InitParams"  [override]
        def get_model(self, key: PRNGKeyArray) -> Model:
                            ^~~~~~~~~~~~~~~~~
train.py:669:25: note: This violates the Liskov substitution principle
train.py:669:25: note: See https://mypy.readthedocs.io/en/stable/common_issues.html#incompatible-overrides
train.py: note: In member "get_initial_model_carry" of class "ZbotWalkingTask":
train.py:785:5: error: Signature of "get_initial_model_carry" incompatible with supertype "ksim.task.rl.RLTask"  [override]
        def get_initial_model_carry(self, rng: PRNGKeyArray) -> tuple[Array, Array]:
        ^
train.py:785:5: note:      Superclass:
train.py:785:5: note:          def get_initial_model_carry(self, model: Any, rng: Array) -> Any | None
train.py:785:5: note:      Subclass:
train.py:785:5: note:          def get_initial_model_carry(self, rng: Array) -> tuple[Array, Array]
train.py: note: At top level:
train.py:822:9: error: Unexpected keyword argument "epochs_per_log_step" for "ZbotWalkingTaskConfig"  [call-arg]
            ZbotWalkingTaskConfig(
            ^
train.py:822:9: error: Unexpected keyword argument "render_full_every_n_seconds" for "ZbotWalkingTaskConfig"  [call-arg]
            ZbotWalkingTaskConfig(
            ^
convert.py: note: In function "main":
convert.py:38:20: error: No overload variant of "load_ckpt" of "TrainMixin" matches argument types "Path", "str"  [call-overload]
        model: Model = task.load_ckpt(ckpt_path, part="model")[0]
                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
convert.py:38:20: note: Possible overload variants:
convert.py:38:20: note:     def load_ckpt(self, path: Path, init_params: InitParams, *, part: Literal['all']) -> tuple[list[Any], list[Any], list[Any], State, ZbotWalkingTaskConfig]
convert.py:38:20: note:     def load_ckpt(self, path: Path, init_params: InitParams, *, part: Literal['model_state_config']) -> tuple[list[Any], State, ZbotWalkingTaskConfig]
convert.py:38:20: note:     def load_ckpt(self, path: Path, init_params: InitParams, *, part: Literal['model']) -> list[Any]
convert.py:38:20: note:     def load_ckpt(self, path: Path, init_params: InitParams, *, part: Literal['opt']) -> list[Any]
convert.py:38:20: note:     def load_ckpt(self, path: Path, init_params: InitParams, *, part: Literal['opt_state'], model: Any | None = ..., optimizer: Any | None = ...) -> list[Any]
convert.py:38:20: note:     def load_ckpt(self, path: Path, init_params: InitParams, *, part: Literal['state']) -> list[State]
convert.py:38:20: note:     def load_ckpt(self, path: Path, init_params: InitParams, *, part: Literal['config']) -> list[ZbotWalkingTaskConfig]
convert.py:74:17: error: Unexpected keyword argument "num_joints" for "export_fn"  [call-arg]
        init_onnx = export_fn(
                    ^
/home/xiaowen/anaconda3/envs/ksim/lib/python3.11/site-packages/kinfer/export/jax.py: note: "export_fn" defined here
convert.py:74:17: error: Unexpected keyword argument "carry_shape" for "export_fn"  [call-arg]
        init_onnx = export_fn(
                    ^
convert.py:80:17: error: Unexpected keyword argument "num_joints" for "export_fn"  [call-arg]
        step_onnx = export_fn(
                    ^
convert.py:80:17: error: Unexpected keyword argument "carry_shape" for "export_fn"  [call-arg]
        step_onnx = export_fn(
                    ^
convert.py:86:20: error: Unexpected keyword argument "joint_names" for "pack"  [call-arg]
        kinfer_model = pack(
                       ^
/home/xiaowen/anaconda3/envs/ksim/lib/python3.11/site-packages/kinfer/export/serialize.py: note: "pack" defined here
convert.py:86:20: error: Unexpected keyword argument "carry_shape" for "pack"  [call-arg]
        kinfer_model = pack(
                       ^
Found 29 errors in 2 files (checked 2 source files)
